index.html

```diff
@@ -61,10 +61,8 @@ <h1 id="title">freeCodeCamp Survey Form</h1>
           <div>
             <label><input type="radio" id="check3" class="checkbox" name="attribute" value="Not sure">Not sure</label>
           </div>

```

Comment:
Please use semantic HTML tags, this document would help you: [https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md](https://github.com/Feliverse/reviewIvanProject/pull/1/url)
Using semantic HTML improved the review time and understanding from your code partners and readers